scripts/beachball/getScopes.ts

```diff
@@ -0,0 +1,47 @@
+import { AllPackageInfo, getAllPackageInfo } from '../monorepo/index';
+
+/**
+ * Reads package info from the monorepo and generates the scopes for beachball bump and release
+ * Differentiates between vNext and v8 releases
```

Comment:
nit since this will be run together when viewing the comment in intellisense
```suggestion
 * Reads package info from the monorepo and generates the scopes for beachball bump and release.
 * Differentiates between vNext and v8 releases.
```